docs/ios.md

```diff
@@ -105,7 +184,11 @@ load("@rules_player//ios:defs.bzl", "assemble_pod")
 assemble_pod(<a href="#assemble_pod-name">name</a>, <a href="#assemble_pod-podspec">podspec</a>, <a href="#assemble_pod-srcs">srcs</a>, <a href="#assemble_pod-data">data</a>)
 </pre>
 
-Assemble a zip file for a podspec and related sources
+Assemble a zip file for a podspec and related sources.
+
+> [!WARNING]
+> This function is deprecated. Use `assemble_ios_release` instead.
```

Comment:
Should we give a timeline for its full deprecation? IMO we can add it to the list for things to do for the next major